tareas/clase-6/tarea 1, clase 6.js

```diff
@@ -0,0 +1,93 @@
+let miembrosGrupoFamiliar = Number(prompt('Cuantos miembros hay en el grupo familiar'));
+
+    for (i = 0; i< miembrosGrupoFamiliar; i++) {
+        CrearMiembros(i);
+    }
+
+document.querySelector('#calcular').onclick = function() {
+    let edades = SacarEdades ();
+    document.querySelector('#mayor-edad').value = sacarmayorEdad ();
+    document.querySelector('#menor-edad').value = sacarmenorEdad ();
+    document.querySelector('#promedio-edad').value = sacarpromedioEdad(edades);
+
+    return false;
+};
+
+document.querySelector('#reset').onclick = function () {
+    borrarIntegrantes ();
+    borrarnumMayor ();
+    borrarnumMenor ();
+    borrarPromedio ();
+
+    return false
+};
+ 
+function CrearMiembros(numero = 1) {
+
+    if (numero >= 1) {
+        acumulador = 0;
+        acumulador + 1;
+    }
+    const div = document.createElement('div');
+    div.className = 'integrante';
+
+    const label = document.createElement('label');
+    label.textContent = 'Edad del integrante Nro: ' + (acumulador);
+
+    const input = document.createElement('input');
+    input.type = 'number';
+
+    div.appendChild(label);
+    div.appendChild(input);
+
+    const miembros = document.querySelector('#miembros');
+    miembros.appendChild(div);
+}
+
+function SacarEdades () {
+    const edadesinputs = document.querySelectorAll('.integrante input');
+    const edadesMiembros = [];
+    for (let j = 0; j < edadesinputs.length; j++) {
+        edadesMiembros.push(Number(edadesinputs[j].value));
+    }
+    return edadesMiembros;
+}
+
+function sacarmayorEdad () {
+    let mayorEdad = SacarEdades ();
+    let resultadomayorEdad = Math.max(...mayorEdad);
```

Comment:
Esto está bien, pero te recomiendo que lo resuelvas con un `for` para pensarlo vos sólo.